app/src/main/res/values/strings.xml

```diff
@@ -1,8 +1,9 @@
 <resources>
     <string name="app_name" translatable="false">Template</string>
     <!--TODO: Replace base url with the correct url-->
-    <string name="base_url" translatable="false">https://xmartlabs.com/</string>
-
+    <string name="base_url" translatable="false">https://taskloans-staging.herokuapp.com/api/v1/</string>
+    <string name="base_task_url" translatable="false">https://taskloans-staging.herokuapp.com/api/v1/tasks/1/users</string>
```

Comment:
we ussually don't put compleate urls here we tend to use the base and add constants to them in the Service declarations